src/game/Entities/Player.cpp

```diff
@@ -7566,6 +7566,9 @@ void Player::RemovedInsignia(Player* looterPlr)
     if (!corpse)
         return;
 
+    WorldPacket data(SMSG_PLAYER_SKINNED, 0);
+    GetSession()->SendPacket(data);
```

Comment:
This packet also has an unknown uint8 packet fill in vanilla client. Fill it with 0 and maybe test what filling it with 1 does.

Reply:
Yes my bad, vmangos fills with 0.